x-pack/test/functional/es_archives/security_solution/ecs_fields_duplicated_for_alerts/data.json

```diff
@@ -0,0 +1,44 @@
+{
+  "type": "doc",
+  "value": {
+    "id": "1",
+    "index": "ecs_fields_duplicated_for_alerts",
+    "source": {
```

Comment:
FYI this new archive reads a little wonky since you'd expect the `data_stream` fields to be in here, right? But because they're `constant_keyword` fields they are defined on the mappings themselves, and not the documents.